src/pages/BDI_BatchOverride.page

```diff
@@ -0,0 +1,56 @@
+<apex:page standardController="DataImportBatch__c" extensions="BDI_BatchOverride_CTRL" showHeader="true" applyHtmlTag="true" applyBodyTag="true"
+    docType="html-5.0"
+    standardStylesheets="false"
+>
+    <!-- we need this html tag to get the SVG to work in visualforce.  but we also need the applyHtmlTag=true to allow showHeader=true.
+    one would think this wouldn't work, but somehow it is allowed and makes the SVG work! -->
+
+    <html xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <apex:stylesheet value="{!URLFOR($Resource.CumulusStaticResources, '/npsp-slds/npsp-common.css')}" />
+
+        <apex:slds />
+        <div class="slds-scope">
+        <apex:form id="idForm" html-novalidate="novalidate" >
+
+            <!-- PAGE HEADER -->
+            <div class="slds-page-header">
+                <div class="slds-grid">
+                    <div class="slds-col slds-has-flexi-truncate">
+                        <div class="slds-media slds-no-space slds-grow">
+                            <div class="slds-media__figure">
+                              <span class="slds-icon_container slds-icon-custom-custom18">
+                                <svg class="slds-icon" aria-hidden="true">
+                                  <use xlink:href="{!URLFOR($Asset.SLDS, 'assets/icons/custom-sprite/svg/symbols.svg#custom17')}" ></use>
```

Comment:
The SVGs have an issue in Edge when in Lightning and in IE11, I switched to using the .pngs instead. I also created a header component to just pass the png name and button actions for reusability but it's not merged yet, jess is currently testing it.